security/identity-and-access-management/oracle-identity-governance/webservice-samples/extend-access-ws/README.md

```diff
@@ -0,0 +1,70 @@
+# Extend Access WebService
+
+This asset contains the code and deployment items for a RESTful service which will extend user access rights.
+
+The scheduled task needs to be used in conjunction with the Access Extension Notification scheduled task, which will generate the data and send the required actionable notification links. The endpoints exposed by this WebService will act as backend providers for those links.
+
+Developed on and compatible with OIG 11g R2 PS3 and above.
+
+## When to use this asset?
+
+When there's a need to provide or demonstrate the functionality described above or something similar, which can be adapted from the provided code.
+
+## How to use this asset?
+
+### Pre-requisites and dependencies
+
+The WebService is build using the JAX-RS specification and API.
+
+As such, the following jar files are required as dependencies and need to be used during the build process:
```

Comment:
Please add a sample command line for compiling the code.

Reply:
Typically one would use Ant/Maven here. I've added a line to that extent with a manual war file build example as well.